microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,148 @@
+
+## Edit and resend network requests more reliably in the Network Console tool
+
+<!-- RANK 3 -->
+<!-- Title: Edit and resend network requests more reliably in the Network Console tool  -->
+<!-- Subtitle: Modify and resend network requests that have been logged in the Network tool with the Network Console tool. -->
+
+Previously in Microsoft Edge, selecting **Edit and resend** for a network request from the Network tool may not have reliably opened the Network Console tool and prepopulated it with the values from the request. 
+
+In Microsoft Edge 105, this issue has been fixed. The **Edit and resend** now prepopulates the **Network Console** tool with the values for the network request that you are looking to resend. You can continue to modify these values before sending the request from the **Network Console** tool. Thank you for sending us your feedback about this issue!
+
+Right-click the network request that you want to change and resend, and then select **Edit and Resend**.
+
+![Edit and Resent](./devtools-105-images/edit-and-resend.png)
+
+In the Network Console, edit the network request information, and then click the *Send* button.
+
+![Network Console Edit and Send](./devtools-105-images/networkconsole-edit.png)
+
+See also:
+* [Compose and send web API requests using the Network Console tool](/microsoft-edge/devtools-guide-chromium/network-console/network-console-tool#starting-from-the-network-tool)
+
+
+<!-- ====================================================================== -->
+## Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View
+
+<!-- Title: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View -->
+<!-- Subtitle: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. -->
+
+Several improvements to [Focus Mode](/devtools-guide-chromium/experimental-features/focus-mode) in existing location controls for DevTools have been made, including a new control for changing the orientation of **Quick View**.
+
+The Settings menu now provides options for changing DevTools between docked locations and a separate window. This menu also contains settings for how the **Activity Bar** is displayed. Starting in Edge 105, these options can be navigated and changed more quickly. The previous submenus have been replaced with toggle buttons. Additionally, improved contrast means the option you have selected is more easily visible at a glance.
+
+Before:
+
+![Focus Mode Before](./devtools-105-images/before-docking-menu.png) 
+
+After:
+
+![Focus Mode After](./devtools-105-images/after-docking-menu.png)
+
+You can now change the orientation of the **Quick View** panel as well. To change Quick View to a vertical split, select the location toggle (**Dock Quick View to the right**). Selecting this toggle a second time will return **Quick View** to a horizontal split.
+
+![Quick View from Console](./devtools-105-images/quickview-console.png) 
```

Comment:
This image does not seem correct. I would recommend drawing a red box not around Quick View but the toggle button for the orientation: 
<img width="961" alt="image" src="https://user-images.githubusercontent.com/34316555/187559658-194fe9c6-b7ed-4e39-bdff-9b169e381b9a.png">